CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.26.0
```

Comment:
Can we make the changelog items more descriptive? 

something like: 
`Enable start and stop ordering for containers in a task`
`Container level configurable start and stop container timeouts`

not sure what this is : `Shutdown order is now observed`